tox.ini

```diff
@@ -5,13 +5,7 @@ envlist = py26,py27,py33,py34
 # This is helpful to test installation but takes extra time
 skipsdist = True
 
-[testenv:py26]
-# Python 2.6 requires two extra test dependencies
-deps =
-    unittest2
-    -rrequirements.txt
-
 [testenv]
-deps = -rrequirements.txt
 commands =
-    nosetests {posargs:tests/unit}
+    {toxinidir}/scripts/ci/install
```

Comment:
That sounds good. I like it. And if we do go with this route, we may also want to adjust this PR's title to reflect the new approach.